code/game/gamemodes/miniantags/abduction/machinery/dispenser.dm

```diff
@@ -8,77 +8,77 @@
 	var/list/gland_types
 	var/list/gland_colors
 	var/list/amounts
-
-/obj/machinery/abductor/gland_dispenser/proc/random_color()
-	//TODO : replace with presets or spectrum
-	return rgb(rand(0,255),rand(0,255),rand(0,255))
+	var/list/color_pool = list(
+		"#700b08",
+		"#b87411",
+		"#8e8004",
+		"#2d7248",
+		"#095da2",
+		"#4d238e",
+		"#661553",
+		"#a70b03",
+		"#576402",
+		"#204a77",
+		"#994225",
+		"#663f36",
+		"#bd3675"
+	)
 
 /obj/machinery/abductor/gland_dispenser/Initialize(mapload)
 	. = ..()
 	gland_types = subtypesof(/obj/item/organ/internal/heart/gland)
 	gland_types = shuffle(gland_types)
 	gland_colors = new/list(gland_types.len)
 	amounts = new/list(gland_types.len)
-	for(var/i=1,i<=gland_types.len,i++)
-		gland_colors[i] = random_color()
+	for(var/i in 1 to length(gland_types))
+		gland_colors[i] = pick_n_take(color_pool)
```

Comment:
Is it possible that the number of glands might be larger than the pool of colors? I think you might be better off making a shuffled version of the list (shuffle()) and iterating it vs pick_n_taking

Reply:
I might actually add a few more colors to the color pool, as there has been some interest in adding more glands to those available to abductors, but I should probably consider doing both at this point as I am not aware of how many exactly will be added. I will look into this.

Reply:
Updated with shuffle and added some new colors as well in the latest commit.